README.md

```diff
@@ -1,182 +1,176 @@
 ```
 
-## Specifying a library
+These and other examples are discussed in more detail in the 
+[Importing from Modules](articles/import.html#importing-functions-from-module-scripts) 
+section of the package vignette.
+
+### Choosing where import looks for packages or modules
 
 The `import` package will by default only use the latest specified library
 (i.e. the result of `.libPaths()[1L]`). It is possible to specify a different
 library using the `.library` argument in any of the `import` functions.
 One import call can only use *one* library so there will not be ambiguity
 as to where imports come from.
 
-## Using .R scripts as "modules" (currently only in GitHub version)
-
-The `import` package allows for importing objects defined in script files,
-which we will here refer to as "modules".
-The module will be fully evaluated by `import` when an import is requested, 
-after which objects such as functions or data can be imported. 
-Such modules should be side-effect free, but this is
-not enforced. Attachments are detached (e.g. packages attached by `library`)
-but loaded namespaces remain loaded. This means that *values* created 
-by functions in an attached namespace will work with `import`, but 
-functions to be exported *should not* rely on such functions (use function
-importing in the modules instead).
-
-If a module is modified, `import` will
-realize this and reload the script if further imports are executed or 
-re-executed; otherwise additional imports will not cause the script to be 
-reloaded for efficiency. As the script is loaded in its own environment 
-(maintained by `import`) dependencies are kept (except those exposed through
-attachment), as the following small example shows.
-
-**Contents of "some_module.R":**
-```R
-## Possible
-library(ggplot2)
-
-## But would be better:
-#import::here(qplot, .from = ggplot2)
-
-## Note this operator overload is not something you want to `source`!
-`+` <- function(e1, e2)
-  paste(e1, e2)
+When importing from a module (.R file), the directory where `import` looks for
+the module script can be speecified with the with `.directory` paremeter. 
+The default is `.` (the current working directory).
 
-## Some function relying on the above overload:
-a <- function(s1, s2)
-  s1 + rep(s2, 3)
+### Choosing where the imported functions are placed
 
-## Another value.
-b <- head(iris, 10)
+One can also specify which names to use in the search path and use several to 
+group imports:
 
-## A value created using a function exposed by attachment 
-p <- qplot(Sepal.Length, Sepal.Width, data = iris, color = Species)
+```R
+import::from(magrittr, "%>%", "%$%", .into = "operators") 
+import::from(dplyr, arrange, .into = "datatools")
+```
 
-## A function relying on a function exposed through attachment:
-plot_it <- function()
-  qplot(Sepal.Length, Sepal.Width, data = iris, color = Species)
+If using pied-à-terres actively, one might prefer the alternative syntax 
```

Comment:
I think this word, and reasoning, was removed above. Maybe use a more standard word, or reintroduce it.

Reply:
Added a short intro before this usage.